papers/Raku_Steering_Council_Code.md

```diff
@@ -1,39 +1,31 @@
 Abstract
 ========
 
-This document proposes a model of Raku governance based around a steering
-council. The council has broad authority, which they seek to exercise
-as rarely as possible; instead, they use this power to establish
-standard processes.  It follows the general philosophy that it's better
-to split up large changes into a series of small changes that can be
-reviewed independently: instead of trying to do everything in one proposal,
-focus is on providing a minimal-but-solid foundation for further governance
-decisions.
+This document proposes a model of Raku governance based around a Steering
+Council, a core development team and a Raku users' community. 
+The Steering Council has broad authority, which it should exercise
+as rarely as possible. Instead, the Steering Council uses this power to establish
+standard processes.
 
 Rationale
 =========
 
 The main goals of this proposal are:
 
-* Be **boring**: We're not experts in governance, and we don't think
-Raku is a good place to experiment with new and untried governance
-models. So this proposal sticks to mature, well-known, previously
-tested processes as much as possible. The high-level approach of a
-mostly-hands-off council is arguably the most common across large
-successful F/OSS projects, and low-level details are derived
-directly from Python's governance, which was influenced by Django's
-governance.
+* Be **boring**: The proposal is based on well-known, previously
+tested processes. The high-level approach of a
+mostly-hands-off council has been applied in large
+successful F/OSS projects.
 * Be **simple**: Pare things down to the minimum needed to make it
-workable: the council, the core team (who elect the council), and
-the process for changing the document. The goal is Minimum Viable
-Governance.
-* Be **comprehensive**: But for the things that need to be defined,
-it should cover all the bases.  Having a clear and unambiguous
-set of rules also helps minimize confusion and resentment.
-* Be **flexible and light-weight**: It will take time and experimentation
-to find the best processes for working together. By keeping this
-proposal as minimal as possible, maximal flexibility for adjusting
-things later is retained, while minimizing the need for heavy-weight
+workable: the council, the core team (who elect the council), a Code of Conduct, and
```

Comment:
I wouldn't mention a Code of Conduct here.  Please don't get me wrong: I am in favour of one.  But adding it here and now is going to complicate matters.